server/facets/create.ts

```diff
@@ -28,7 +29,7 @@ const getSequelizePropsDefinition = (props: FacetProps) => {
 };
 
 export const createSequelizeModelsFromFacetDefinitions = (sequelize: Sequelize) => {
-	const FacetBinding = sequelize.import('./models/facetBinding') as any;
+	// const FacetBinding = sequelize.import('./models/facetBinding') as any;
```

Comment:
Why was this commented out? Was it unused/unnecessary?

Reply:
ah sorry, that should have been removed

you cannot use `sequelize.import` anymore in v6, so its now defined with sequelize.define in `models/facetBingding`